client/src/www/views/root_view/add_access_key_dialog/index.ts

```diff
@@ -105,13 +112,22 @@ export class AddAccessKeyDialog extends LitElement {
         </md-text-button>
         <md-filled-button
           @click=${this.handleConfirm}
-          ?disabled=${!this.accessKey || !this.isValidAccessKey(this.accessKey)}
+          ?disabled=${!this.accessKey || !this.isValidAccessKey}
```

Comment:
The validation is `async`, will the user be able to click this button even if the key is invalid during validation? Take an example of the following sequence:

```
Key is valid, <Button Enabled>, but user wants to change the key
Paste in the new (invalid) key
Validating ...
<Button still Enabled>    <= is the user able to click the button here?
Validation done, invalid key!
<Button Disabled>
```

Reply:
They will, but the add will fail.